package.json

```diff
@@ -5,13 +5,13 @@
   "main": "index.js",
   "scripts": {
     "test": "NODE_ENV=test jest --coverage",
-    "start:dev": "webpack-dev-server --mode development --config config/webpack.base.config.js --hot --history-api-fallback",
+    "dev": "webpack-dev-server --mode development --config config/webpack.base.config.js --hot --history-api-fallback",
     "prestart:prod": "webpack --mode production --config config/webpack.prod.config.js --env.NODE_ENV=production --progress",
     "start:prod": "node server",
     "lint": "eslint --format codeframe src/"
   },
   "engines": {
-    "node": "8.11"
+    "node": "10.20.1"
```

Comment:
i guess we can use 16.x without any troubles

Reply:
@VladWithCookies I use 10.20.1 as default. If it's not a problem I'll live it